Resources/Locale/en-US/language/languages.ftl

```diff
@@ -61,6 +61,9 @@ language-RobotTalk-description = A language consisting of harsh binary chirps, w
 language-Sign-name = Tau-Ceti Basic Sign Language
 language-Sign-description = TCB-SL for short, this sign language is prevalent among mute and deaf people.
 
+language-Marish-name = Marish
+language-Marish-description = An Language that can be used to speak in Empathy, Sharing eachother emotions with only one word, Shadowkins speaks this language with ease, tho its is nearly impossible to replicate it or learn it.
```

Comment:
I understand Trystan's first language isn't English but two native English speakers (at least) reviewed and approved this PR.